META.json

```diff
@@ -0,0 +1,44 @@
+{
+   "name": "griddb_fdw",
+   "abstract": "Foreign Data Wrapper for GridDB databases",
+   "description": "PostgreSQL extension which implements a Foreign Data Wrapper (FDW) for GridDB.",
+   "version": "0.0.1",
+   "maintainer": "pgspider",
+   "license": "",
+   "provides": {
+      "sgriddb_fdw": {
+         "abstract": "Foreign Data Wrapper for GridDB databases",
+         "file": "griddb_fdw.c",
+         "docfile": "README.md",
+         "version": "0.0.1"
+      }
+   },
+   "prereqs": {
+      "runtime": {
+         "requires": {
+            "PostgreSQL": "10.0.0"
```

Comment:
This FDW supports from version 11.
So clould you update the PostgreSQL required version?

Reply:
Fixed.

Reply:
Thank you for support.